lib/launcher.js

```diff
@@ -351,11 +351,16 @@ class Launcher {
 
     /**
      * emit event from child process to reporter
-     * @param  {Object} m  event object
+     * @param  {String} cid
+     * @param  {Object} m event object
      */
-    messageHandler (m) {
+    messageHandler (cid, m) {
```

Comment:
Might I suggest perhaps a different PR would be a good place to standardize on either `rid` or `cid` (probably `cid`)?